docs/sources/http_api/user.md

```diff
@@ -69,6 +69,38 @@ parent = "http_api"
       "isGrafanaAdmin": true
     }
 
+## Get single user by Username(login) or Email
+
+    `GET /api/users/loginoremail/:loginOrEmail`
+
+    **Example Request using the email option**:
+
+        GET /api/loginoremail/user@mygraf.com HTTP/1.1
```

Comment:
Thanks @torkelo. Indeed the lookup url looks better. I'll have a look into it and get back to you soon.